plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/AbstractKuduWithStandardInferSchemaConnectorTest.java

```diff
@@ -32,7 +36,13 @@
     @Test
     public void testListingOfTableForDefaultSchema()
     {
-        assertEquals(computeActual("SHOW TABLES FROM default").getRowCount(), 0);
+        // The special $schemas table is created when listing schema names with schema emulation enabled
+        // Depending on test ordering, this table may or may not be created when this test runs, so filter it out
+        List<MaterializedRow> rows = computeActual("SHOW TABLES FROM default").getMaterializedRows()
+                .stream()
+                .filter(row -> !((String) row.getField(0)).contains("$schemas"))
```

Comment:
"if the name is fixed" can you explain what you mean? I'm not following.

Reply:
I believe that since the table we exclude is always called "$schema" we can just make it part of query like `SHOW TABLES FROM default NOT LIKE '$schema'` or maybe use one of the assertions like `assertThat(row).containsExactlyInAnyOrderElementsOf(<row created by hand>)`. (search for `containsExactlyInAnyOrder` to see examples).